query/outputnode.go

```diff
@@ -150,6 +150,68 @@ func newEncoder() *encoder {
 	return e
 }
 
+// Sort the given fastJson list
+func (enc *encoder) MergeSort(headRef *fastJsonNode) {
+	head := *headRef
```

Comment:
I think it would be good if all the other functions' names are not be capitalized apart from `MergeSort`.